src/main/resources/application-dev.yml

```diff
@@ -24,5 +24,10 @@ spring:
         jwt:
           public-key-location: classpath:local-public-key.pub
 
+logging:
+  level:
+    uk:
+      gov: DEBUG
+
 server:
```

Comment:
We were running at debug by default, we want to run at info by default and override that on the dev profile